content/id/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,116 @@
+untuk informasi lebih lanjut tentang cara kerja keanggotaan dalam komunitas Kubernetes.
+
+Selebihnya dari dokumen ini akan menguraikan beberapa cara unik dari fungsi peranan tersebut dalam
+SIG Docs, yang bertanggung jawab untuk memelihara salah satu aspek yang paling berhadapan dengan publik
+dalam Kubernetes - situs web dan dokumentasi dari Kubernetes.
+
+
+<!-- body -->
+
+## Ketua umum (_chairperson_) SIG Docs {#ketua-umum-sig-docs}
+
+Setiap SIG, termasuk SIG Docs, memilih satu atau lebih anggota SIG untuk bertindak sebagai
+ketua umum. Mereka merupakan kontak utama antara SIG Docs dan bagian lain dari
+organisasi Kubernetes. Mereka membutuhkan pengetahuan yang luas tentang struktur
+proyek Kubernetes secara keseluruhan dan bagaimana SIG Docs bekerja di dalamnya. Lihatlah
+[Kepemimpinan (_leadership_)](https://github.com/kubernetes/community/tree/master/sig-docs#leadership)
+untuk daftar ketua umum yang sekarang.
+
+## Tim dan automasi dalam SIG Docs
+
+Otomasi dalam SIG Docs bergantung pada dua mekanisme berbeda:
+Tim GitHub dan berkas OWNERS.
+
+### Tim GitHub
+
+Terdapat dua kategori tim dalam SIG Docs [tim (_teams_)](https://github.com/orgs/kubernetes/teams?query=sig-docs) dalam GitHub:
+
+- `@sig-docs-{language}-owners` merupakan pemberi persetujuan (_approver_) dan pemimpin (_lead_)
+- `@sig-docs-{language}-reviewers` merupakan pengulas (_reviewer_)
+
+Setiap tim dapat di referensikan dengan `@name` mereka dalam komentar GitHub untuk berkomunikasi dengan setiap orang dalam grup.
+
+Terkadang tim Prow dan GitHub tumpang tindih (_overlap_) tanpa kecocokan sama persis. Untuk penugasan masalah, _pull request_, dan untuk mendukung persetujuan PR,
+otomatisasi menggunakan informasi dari berkas `OWNERS`.
+
+
+### Berkas OWNERS dan bagian yang utama (_front-matter_)
+
+Proyek Kubernetes menggunakan perangkat otomatisasi yang disebut prow untuk melakukan otomatisasi
+yang terkait dengan masalah dan _pull request_ dalam GitHub.
+[Repository website Kubernetes](https://github.com/kubernetes/website) menggunakan
+dua buah [prow _plugin_](https://github.com/kubernetes/test-infra/tree/master/prow/plugins):
+
+- blunderbuss
+- approve
+
+Kedua _plugin_ menggunakan berkas
+[OWNERS](https://github.com/kubernetes/website/blob/master/OWNERS) dan
+[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES)
+dalam level teratas dari repository GitHub `kubernetes/website` untuk mengontrol
+bagaimana prow bekerja dalam repositori.
+
+Berkas OWNERS berisi daftar orang-orang yang menjadi pengulas dan pemberi persetujuan dalam SIG Docs.
+Berkas OWNERS juga bisa terdapat dalam subdirektori, dan dapat menimpa peranan karena 
+dapat bertindak sebagai pengulas atau pemberi persetujuan berkas dalam subdirektori itu dan
+apa saja yang ada dibawahnya. Untuk informasi lebih lanjut tentang berkas PEMILIK pada umumnya, lihatlah
```

Comment:
I don't think OWNERS should be translated since it's name of file/automation
```suggestion
apa saja yang ada di dalamnya. Untuk informasi lebih lanjut tentang berkas OWNERS pada umumnya, lihatlah
```

Reply:
Ok!